README.md

```diff
@@ -194,6 +194,27 @@ tfrecorder inspect \
     --output_dir=/tmp/output
 ```
 
+### Creating an image CSV file
+
```

Comment:
Do we want to note how the labels and splits are generated here?

Reply:
Makes sense. Maybe we could just reference the image directory section for the schema.